charts/zenko/values.yaml

```diff
@@ -95,4 +95,25 @@ redis-ha:
     create: true
   replicas:
     servers: *nodeCount
-    sentinels: *nodeCount
\ No newline at end of file
+    sentinels: *nodeCount
+
+grafana:
+  enabled: false
+  # When 'grafana.enabled' is true, a grafana chart is deployed as part of the Zenko stack
+  ingress:
+    enabled: true
+    hosts:
+      - grafana.local
+  sidecar:
+    datasources:
+      enabled: true
+      # Every config map with the following label will be used as datasource. By default,
+      # promtheus is set as datasource under zenko/templates/datasources.yaml
+      label: grafana-datasource
+    dashboards:
+      enabled: true
+      # Likewise, every config map with the following label will be used as a source for
+      # a dashboard.
+      label: grafana-dasboard
```

Comment:
Please add a dashboard with this PR, to make sure this functionality actually works.